src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+@use "../utils/_variables" as *;
+@use "../utils/_functions" ;
+@use "../utils/_mixins" ;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  padding: $calendar-padding;
+  max-width: 7 * $cell-size + 2 * $calendar-padding + 6 * $cell-gap;
+  width: 100%;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.